stan/math/prim/arr/functor/integrate_ode_rk45.hpp

```diff
@@ -127,7 +127,7 @@ namespace stan {
                                                            double,
                                                            std::vector<double>,
                                                            double>() ),
-                      coupled_system,
+                      boost::ref(coupled_system),
```

Comment:
Not a change request, just a question:  Is this `boost::ref` because you're calling `boost::numeric::odeint::integrate_times` on the coupled system and don't want to make a copy (I see that `integrate_times` just takes the system, not a system reference)?  Does it actually prevent copying?